<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>721</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLabel" name="lab_selectedinfo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="ExtendedGraphicView" name="gv_field">
      <property name="minimumSize">
       <size>
        <width>625</width>
        <height>625</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>625</width>
        <height>625</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>score:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="lab_mouseinfo">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="pte_mapinput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>465</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>200</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_parsemap">
      <property name="text">
       <string>Parse map</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>игра</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>настройки</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>помощь</string>
    </property>
    <addaction name="action_10"/>
    <addaction name="separator"/>
    <addaction name="action_12"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>новая игра</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>сохранить игру</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>перезапустить уровень</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>отменить ход</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
  <action name="action_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>стандартный вариант оформления</string>
   </property>
  </action>
  <action name="action_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>второй вариант оформления</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>посмотреть справку</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>лучшие результаты</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>extendedgraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
